tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -9,6 +9,24 @@ MK_DIR := $(dir $(MK_PATH))
 # Verbose build
 V := 1
 
+ARCH := $(shell uname -m)
```

Comment:
I think we're now building the artifacts for the CI in VMs, so I think this ARCH will always be x86_64. I guess this conditional won't hurt. All the SE components can be cross-compiled, right?

Reply:
Yes, it is in progress (https://github.com/kata-containers/kata-containers/pull/6586). I have to change the conditional.